crates/holochain/src/core/workflow/publish_dht_ops_workflow.rs

```diff
@@ -78,7 +81,9 @@ pub async fn publish_dht_ops_workflow(
         }
     }
 
-    info!("published {} ops", success.len());
+    if to_publish_count > 0 {
+        info!("published {}/{} ops", success.len(), to_publish_count);
```

Comment:
This workflow needs to run because of the loop logic at the end but logging 0 work done is just noise we don't need